pro/src/components/ActionsBarSticky/ActionsBarSticky.module.scss

```
@@ -57,6 +57,10 @@
 }
 
 @media (min-width: size.$laptop) {
+  .actions-bar {
+    padding-right: size.$main-content-padding;
+  }
+
   .actions-bar-new-interface {
     padding-left: calc(rem.torem(size.$side-nav-width) + rem.torem(32px));
   }
```

Avec le changement actuel, je pense que ça va casser l'ancienne interface.
```suggestion
  .actions-bar-new-interface {
    padding-right: size.$main-content-padding;
    padding-left: calc(rem.torem(size.$side-nav-width) + rem.torem(32px));
  }
```